io_scene_niftools/modules/nif_import/animation/material.py

```diff
@@ -64,7 +64,7 @@ def import_material_controllers(self, n_geom, b_material):
     def import_material_alpha_controller(self, b_material, n_material):
         # find alpha controller
         n_alphactrl = math.find_controller(n_material, NifFormat.NiAlphaController)
```

Comment:
Hope in the future we get to the point where we can process controllers as we find them, rather than doing lookups.

Reply:
It isn't a lookup, it's really just - see if this node has a valid controller attached.

Reply:
Yeah its more something for future work, would like to just process the list of controllers, using the delegate method.